src/diffusers/models/attention_dispatch.py

```diff
@@ -1350,6 +1354,40 @@ def _flash_attention(
     return (out, lse) if return_lse else out
 
 
+@_AttentionBackendRegistry.register(
+    AttentionBackendName.FLASH_HUB,
+    constraints=[_check_device, _check_qkv_dtype_bf16_or_fp16, _check_shape],
+)
+def _flash_attention_hub(
+    query: torch.Tensor,
+    key: torch.Tensor,
+    value: torch.Tensor,
+    dropout_p: float = 0.0,
+    is_causal: bool = False,
+    scale: Optional[float] = None,
+    return_lse: bool = False,
+    _parallel_config: Optional["ParallelConfig"] = None,
+) -> torch.Tensor:
+    if _parallel_config:
```

Comment:
Use:
```python
@_AttentionBackendRegistry.register(
    AttentionBackendName.FLASH_HUB,
    constraints=[_check_device, _check_qkv_dtype_bf16_or_fp16, _check_shape],
    supports_context_parallel=False,
)
```
https://github.com/huggingface/diffusers/blob/544ba677dd97a49c8124208837025aa8b5ab639e/src/diffusers/models/attention_dispatch.py#L1307

It will raise an error when trying to enable parallelism with this backend. This check isn't needed.